EdxMobile/app/navigators/AppNavigator.tsx

```diff
@@ -71,13 +79,17 @@ const AppStack = observer(function AppStack() {
             name="Welcome" 
             component={WelcomeScreen}
           />
+          <Stack.Screen 
+            name="CourseDetail" 
+            component={CourseDetailScreen} 
+          />
         </>
       ) : (
         <>
           <Stack.Screen name="Login" component={LoginScreen} />
         </>
       )}
-      {/** 🔥 Your screens go here */}
+      {}
```

Comment:
I think it goes here instead of the conditional. The conditional renders first.

Reply:
Adding it here seems to cause the app to go straight to the course detail module rather than rendering the log in -> welcome screen first. Is there a workaround?

Reply:
don't worry about this Ill take a look